SabreTools.FileTypes/CHD/CHDFile.cs

```diff
@@ -1,16 +1,19 @@
-﻿using System;
+﻿using System.Collections.Generic;
+using System;
 using System.IO;
 using System.Text;
 
+using SabreTools.Core.Tools;
 using SabreTools.IO;
 
+// TODO: add check
 namespace SabreTools.FileTypes.CHD
 {
     /// <summary>
     /// This is code adapted from chd.h and chd.cpp in MAME
     /// Additional archival code from https://github.com/rtissera/libchdr/blob/master/src/chd.h
     /// </summary>
-    public abstract class CHDFile : BaseFile
+    public class CHDFile : BaseArchive
```

Comment:
Why is this no longer `abstract`? Each of the versioned CHDs still need to be individually accounted for.

Reply:
its now a basearchive, more treated like an archive not a file. Oportunity is to transform in both directions.